docs/content/en/docs/policy-library/observability/_index.md

```diff
@@ -30,95 +33,170 @@ description: >
 ```
 
 ### Example Output
 
 ```shell
-"2023-11-01T02:56:54.926403604Z /usr/bin/bpftool prog list programType:BPF_PROG_TYPE_SOCKET_FILTER programInsn:2"
+"2023-10-31T18:44:22.777962637Z default/xwing /tmp/nc ebpf.io 1234"
 ```
 
-## Kernel Module Audit Trail {#kernel-module}
+## sudo Invocation Monitoring {#sudo}
 
 ### Description
 
-Audit loading of kernel modules
+Monitor sudo invocations
 
 ### Use Case
 
-Understanding exactly what kernel modules are running in the cluster is crucial to understand attack surface and any malicious actors loading unexpected modules.
+sudo is used to run executables with particular privileges. Creating a audit log of sudo invocations is a common best-practice.
 
 ### Policy
 
-[modules.yaml](https://raw.githubusercontent.com/cilium/tetragon/main/examples/policylibrary/modules.yaml)
+No policy needs to be loaded, standard process execution observability is sufficient.
 
 ### Example jq Filter
 
 ```shell
- jq 'select(.process_kprobe != null) | select(.process_kprobe.function_name | test("security_kernel_module_request"))  | "\(.time) \(.process_kprobe.process.binary) \(.process_kprobe.process.arguments) module:\(.process_kprobe.args[0].string_arg)"'
+jq 'select(.process_exec != null) | select(.process_exec.process.binary | contains("sudo")) | "\(.time) \(.process_exec.process.pod.namespace) \(.process_exec.process.binary) \(.process_exec.process.arguments)"'
 ```
 
 ### Example Output
 
 ```shell
-"2023-11-01T04:11:38.390880528Z /sbin/iptables -A OUTPUT -m cgroup --cgroup 1 -j LOG module:ipt_LOG"
+"2023-10-31T19:03:35.273111185Z null /usr/bin/sudo -i"
 ```
 
-## Shared Library Loading {#library}
+## Privileges Escalation via SUID Binary Execution {#privileges-suid}
 
 ### Description
 
-Monitor loading of libraries
+Monitor execution of SUID "Set User ID" binaries.
 
 ### Use Case
 
-Understanding the exact versions of shared libraries that binaries load and use is crucial to understand use of vulnerable or deprecated library versions or attacks such as shared library hijacking.
+The "Set User Identity" and "Set Group Identity" are permission flags. When set on a binary file, the binary will execute with the permissions
+of the owner or group associated with the executable file, rather than the user executing it. Usually it is used to run programs with elevated privileges to perform specific tasks.
 
-### Policy
+Detecting the execution of `setuid` and `setgid` binaries is a common
+best-practice as attackers may abuse such binaries, or even create them
+during an exploit for subsequent execution.
 
-[library.yaml](https://raw.githubusercontent.com/cilium/tetragon/main/examples/policylibrary/library.yaml)
+### Requirement
 
+Run Tetragon with `enable-process-creds` setting set to enable visibility
```

Comment:
This configuration level doesn't belong to the "Policy Library" section. Can you please point just a reference here to the documentation where you can enable credentials in processX events?

Reply:
> This level of configuration details doesn't belong to the "Policy Library" section. Can you please point just a reference here to the documentation where you can enable credentials in processX events?

I understand but this adds more work as right now we don't have this section, I will try to hack something up but don't want to delay more the PR.

Reply:
I added configuration section hope it's ok ;-)